pkg/config/stepmeta.go

```diff
@@ -385,7 +385,7 @@ func (container *Container) commonConfiguration(keyPrefix string, config *map[st
 	}
 	putStringIfNotEmpty(*config, keyPrefix+"Workspace", container.WorkingDir)
 	putSliceIfNotEmpty(*config, keyPrefix+"Options", OptionsAsStringSlice(container.Options))
-	//putSliceIfNotEmpty(*config, keyPrefix+"VolumeBind", volumeMountsAsStringSlice(container.VolumeMounts))
+	putSliceIfNotEmpty(*config, keyPrefix+"VolumeBind", volumeMountsAsStringSlice(container.VolumeMounts))
```

Comment:
Can't we put it into the `dockerOptions`? Ecspecially if we limit it to one empty volume.